openshift-examples/keycloak.yaml

```diff
@@ -4,39 +4,39 @@ metadata:
   name: keycloak
   namespace: openshift-config
   annotations:
-    description: An example Keycloak server with HTTPS
+    description: An example template for trying out Keycloak on OpenShift
     iconClass: icon-sso
     openshift.io/display-name: Keycloak
     tags: keycloak
-    version: 9.0.4-SNAPSHOT
+    version: 17.0.0-SNAPSHOT
 objects:
   - apiVersion: v1
     kind: Service
     metadata:
       annotations:
-        description: The web server's https port.
+        description: The web server's http port.
       labels:
         application: '${APPLICATION_NAME}'
       name: '${APPLICATION_NAME}'
     spec:
       ports:
-        - port: 8443
-          targetPort: 8443
+        - port: 8080
+          targetPort: 8080
```

Comment:
I don't think I agree. It is always to give secure examples for use. Can we update by keeping secure port working?

Reply:
Please keep in mind these are Quick Starts, not a production guide. We're using `edge` in other examples too, and align this example now to work like the others. see e.g. https://raw.githubusercontent.com/keycloak/keycloak-quickstarts/latest/kubernetes-examples/keycloak.yaml 

For making kc ready for production, we have https://www.keycloak.org/server/configuration-production now.